### ISSUE TITLE
Get prefix from ledger

### ISSUE THREAD

**Issue by emschwartz** [interledgerjs/ilp-plugin-bells#21]
We shouldn't need to configure plugin bells with the prefix because it should be able to get it from the ledger

**Comment by justmoon** [comment]
@cdmcnamara just asked for this, @bensharaf can you do this today?

Added interledger/five-bells-ledger#308 which this depends on.

**Comment by justmoon** [comment]
Also, behavior should be: When no prefix is provided, get the prefix from the ledger when connecting (we already have a call to the metadata endpoint) and if that also doesn't contain a prefix, we should raise an alarm somehow.

**Comment by sharafian** [comment]
Should this get the prefix once, or every time that getPrefix is called?

**Comment by emschwartz** [comment]
Only once

**Comment by sharafian** [comment]
This is now merged into FBL and Plugin Bells
